meta-resin-common/recipes-support/resin-mounts/resin-mounts/var-log-journal.mount

```diff
@@ -1,8 +1,8 @@
 [Unit]
```

Comment:
Typo in commit log. Should be "prior to this mount" instead of "prior this mount"